README.md

```diff
@@ -6,56 +6,57 @@
 
 ![logo](https://raw.githubusercontent.com/opentok/accelerator-core/master/tokbox-logo.png)
 
-# OpenTok Accelerator Core
+# OpenTok Accelerator Core Web
 
+OpenTok's `Accelerator Core Web` provides an easy integration for OpenTok libraries. Wether you've already built your application or are starting to use OpenTok's libraries, `Accelerator Core Web` helps you implement audio/video communication into your application with rich WebRTC features:
 
-```javascript
-otCore.init(options) =>
-  Multi-Party Audio/Video +
-  Text Chat +
-  Screen Sharing +
-  Annotation +
-  Archiving
-```
+  - Multi-Party Audio/Video Communication
+  - Text Chat
+  - Screen Sharing
+  - Annotation
+  - Archiving
+
+##Quickstart
 
-OpenTok Accelerator Core provides a simple way to integrate real-time audio/video into your web application using the OpenTok Platform.  `Core` also integrates with, manages, and provides a single API for the following accelerator packs:
+ To start using audio/video communication with OpenTok's libraries go to [Using accelerator core](#using-accelerator-core). To implement a more granular and complex management for OpenTok's session you can check [Advanced OpenTok Session Management](#advanced-opentok-session-management). 
+ 
+##Using accelerator core
+
+`OpenTok Accelerator Core` provides a simple way to integrate real-time audio/video into your web application using the OpenTok Platform.  It also integrates with, manages, and provides a single API for the following accelerator packs:
 
  - [Text Chat](https://www.npmjs.com/package/opentok-text-chat)
  - [Screen Sharing](https://www.npmjs.com/package/opentok-screen-sharing)
  - [Annotation](https://www.npmjs.com/package/opentok-annotation)
  - [Archiving](https://www.npmjs.com/package/opentok-archiving)
 
-`Core` also internally maintains the state of your OpenTok session for you.  Calling `otCore.state()` returns an object containing:
-```javascript
-  streams     => All current streams
-  streamMap   => The map of stream ids to publisher/subscriber ids
-  publishers  => All current publishers
-  subscribers => All current subscribers
-  meta        => The count of all current publishers and subscribers by type
-```
-*See an example of the publisher, subscriber, and meta data [below](#pubSubData).*
 
-Sample Applications
--------------------
+###Advanced OpenTok Session Management
```

Comment:
I don't think `advanced` is the right word.  Also, I think this session should still go towards the bottom of the readme because `sdk-wrapper` should be somewhat of an afterthought.